src/telemetry/index.ts

```diff
@@ -1,13 +1,44 @@
 import TelemetryReporter from "@vscode/extension-telemetry";
 import * as vscode from "vscode";
-import { provideSingleton } from "../utils";
+import { getFirstWorkspacePath, provideSingleton } from "../utils";
+import { CommandProcessExecutionFactory } from "../commandProcessExecution";
+import { PythonEnvironment } from "../manifest/pythonEnvironment";
 
 @provideSingleton(TelemetryService)
 export class TelemetryService implements vscode.Disposable {
+  private dbtLoomInstalled = "false";
+  constructor(
+    private pythonEnvironment: PythonEnvironment,
+    private commandProcessExecutionFactory: CommandProcessExecutionFactory,
+  ) {
+    this.isDbtLoomInstalled().then((isInstalled) => {
```

Comment:
since we are dealing with less no of users, this is not re calculated when manifest changes. If needed we can add that as well